src/Microsoft.Health.Fhir.Core/Messages/Bundle/BundleResponse.cs

```diff
@@ -10,13 +10,17 @@ namespace Microsoft.Health.Fhir.Core.Messages.Bundle
 {
     public class BundleResponse
     {
-        public BundleResponse(ResourceElement bundle)
+        public BundleResponse(ResourceElement bundle, BundleResponseInfo info)
         {
             EnsureArg.IsNotNull(bundle, nameof(bundle));
+            EnsureArg.IsNotNull(info, nameof(info));
 
             Bundle = bundle;
+            Info = info;
         }
 
         public ResourceElement Bundle { get; }
+
+        public BundleResponseInfo Info { get; }
```

Comment:
What is this used for? I see it in the unit tests, but are we using this information anywhere else? I'm not thrilled with code that is just for testing.

Reply:
I want to start using this as a source of truth for bundle metrics and exposing some of the parameters send by customers that are only identified during the bundle execution. 

The reason for this is to improve the transparency of bundle execution and just log some final metrics and statistics in a single place, when the bundle ends processing.